2996_reflection/reflection.md

```diff
@@ -4132,11 +4150,11 @@ Add a new subsection of [expr.unary]{.sref} following [expr.delete]{.sref}
 ```
 $reflect-expression$:
    ^^ ::
-   ^^ $qualified-namespace-specifier$
-   ^^ $nested-name-specifier$@~_opt_~@ $template-name$
-   ^^ $nested-name-specifier$@~_opt_~@ $concept-name$
    ^^ $type-id$
    ^^ $id-expression$
+
+$qualified-reflection-name$:
+  $nested-name-specifier$@~_opt_~@ $identifier$
```

Comment:
Do we need a row in `$reflect-expression$` for `^^ $qualified-reflection-name$`?

Reply:
Yeah; Jens reminded me too. Done.

Reply:
Yeah but I did it first! I think. I'm not going to check the timestamps.